crates/shadowsocks-service/src/local/tun/mod.rs

```diff
@@ -27,7 +27,8 @@ cfg_if! {
                  target_os = "linux",
                  target_os = "android",
                  target_os = "windows",
-                 target_os = "freebsd"))] {
+                 target_os = "freebsd",
+                 target_os = "openbsd"))] {
```

Comment:
`tun2` doesn't support openbsd: https://github.com/tun2proxy/rust-tun/blob/5b6a80a859da55a65b323f6dbfcf2893744caaec/src/platform/mod.rs#L20-L51

You could open an PR to `tun2`, they are actively responding.